user-management/src/main/elm/sources/View.elm

```diff
@@ -1,74 +1,160 @@
 module View exposing (..)
 
-------------------------------
--- VIEW --
-------------------------------
-
-
 import ApiCalls exposing (deleteUser)
-
-import DataTypes exposing (PanelMode(..), Model, Msg(..), RoleConf, Roles, StateInput(..), User, Username, Users, UsersConf)
+import DataTypes exposing (Model, Msg(..), PanelMode(..), Provider(..), RoleConf, Roles, StateInput(..), User, Username, Users, UsersConf, providerToString)
 import Dict exposing (keys)
-import Html exposing (Html, a, br, button, div, h2, h3, h4, h5, i, input, p, span, text)
-import Html.Attributes exposing (attribute, class, disabled, href, id, placeholder, required, type_, value)
+import Html exposing (..)
+import Html.Attributes exposing (attribute, class, disabled, href, id, placeholder, required, style, tabindex, type_)
 import Html.Events exposing (onClick, onInput)
 import Init exposing (defaultConfig)
-import List exposing (all, any, filter, map)
```

Comment:
Don't expose functions from List. This will fait to build when you use one (especially map, which conflict with the one from HTML) 

line 233 and 248 this fails to build in 0.19.1 due to conflict use List.map instead

Reply:
You still need to import List, just don't expose anything